shacl12-core/index.html

```diff
@@ -282,14 +282,14 @@ <h3>Terminology</h3>
 				<div class="def">
 					<div class="term-def-header">SHACL Lists</div>
 					<div>
-						<span data-syntax-rule="SHACL-list">A <dfn data-lt="SHACL lists">SHACL list</dfn> in an RDF graph <code>G</code> is an <a>IRI</a> or a <a>blank node</a>
+						<span data-syntax-rule="SHACL-list">A <dfn>SHACL list</dfn> in an RDF graph <code>G</code> is an <a>IRI</a> or a <a>blank node</a>
```

Comment:
Given that plural *and* singular must be in the `<dfn data-lt="..."> (as in `"value|values|property value|property values"` above), then the plural `SHACL lists` should not have been deleted, as in this merged PR, but the singular `SHACL list` should have been added, as --
```suggestion
						<span data-syntax-rule="SHACL-list">A <dfn>SHACL list|SHACL list</dfn> in an RDF graph <code>G</code> is an <a>IRI</a> or a <a>blank node</a>
```